src/os/ogc/filter/ogcspatialformatter.js

```diff
@@ -45,31 +45,32 @@ os.ogc.filter.OGCSpatialFormatter.prototype.format = function(feature) {
     // name/desc does not matter.
     var name = os.xml.escape(/** @type {string} */ (feature.get('title') || 'New Area'));
     var description = os.xml.escape(/** @type {string} */ (feature.get('description')));
+    var id = feature.getId();
```

Comment:
I would recommend calling `os.xml.escape` on the id, in case it contains characters that are invalid in an XML attribute.